vstgui/lib/platform/win32/win32frame.cpp

```diff
@@ -875,7 +875,11 @@ LONG_PTR WINAPI Win32Frame::proc (HWND hwnd, UINT message, WPARAM wParam, LPARAM
 				buttons |= kAlt;
 			if (isTouchActive)
 				buttons |= kTouch;
-			CPoint where (GET_X_LPARAM (lParam), GET_Y_LPARAM (lParam));
+
+			// Win32OptionMenu::popup does ClientToScreen for TrackPopupMenu, must do the opposite for MouseUp.
```

Comment:
I don't see any issues even though this is a really generic handler for MouseUp.

Problem is that Win32OptionMenu::popup is very far from this class and I don't quite know if I should pass the state here or not.

something like "ScreenToClient transform only on Win32OptionMenu::popup MouseUp"...

Reply:
Right so onmouseup transforms - does the step sequencer still work? That’s one spot I think of checking right away. I’m thinking where else will mouse up location

Reply:
Step sequencer seems without issues.

I found an unrelated mouse bug in it. While dragging the step sequencer vector with right mouse button, you can drag the vector "out of bounds" of the step sequencer window. The vector arrow itself does not go out of bounds, but the direction it is pointing is following mouse cursor that went out of bounds.

Reply:
MSEG works without issues. I think this can be merged.

Reply:
That’s not a bug! That’s the behavior we wanted! 

Ok cool I will merge this then we will need another pr on surge of course to update the submodule.